src/coreclr/nativeaot/Runtime/CMakeLists.txt

```diff
@@ -228,20 +228,15 @@ if (NOT CLR_CMAKE_TARGET_ARCH_WASM)
 endif()
 add_definitions(-DFEATURE_BASICFREEZE)
 add_definitions(-DFEATURE_CONSERVATIVE_GC)
-<<<<<<< HEAD
 
 if(CLR_CMAKE_TARGET_UNIX)
-  add_definitions(-DFEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP)
   
   # FEATURE_MANUALLY_MANAGED_CARD_BUNDLES only supported for 64 bit hosts
   if (NOT CLR_CMAKE_TARGET_ARCH_WASM) 
     add_definitions(-DFEATURE_MANUALLY_MANAGED_CARD_BUNDLES)
   endif()
 endif()
```

Comment:
It looks like `DFEATURE_MANUALLY_MANAGED_CARD_BUNDLES` was enabled on Windows as well, so this whole section should be:
```
if (NOT CLR_CMAKE_TARGET_ARCH_WASM) # FEATURE_MANUALLY_MANAGED_CARD_BUNDLES only supported for 64 bit hosts
  add_definitions(-DFEATURE_MANUALLY_MANAGED_CARD_BUNDLES)
endif()
```
(After `add_definitions(-DFEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP)`)